distributed/bokeh/scheduler_html.py

```diff
@@ -93,23 +93,43 @@ def get(self, key):
 
 class CountsJSON(RequestHandler):
     def get(self):
+        scheduler = self.server
+        erred = 0
+        nbytes = 0
+        ncores = 0
+        memory = 0
+        processing = 0
+        waiting = 0
+        waiting_data = 0
+        for ts in scheduler.task_states.values():
```

Comment:
This is a little unfortunate.  I'm curious to see in what other situations we lose constant time diagnostics.

Reply:
AFAIR, diagnostics and bokeh are the only places.